lists/CD-MJDH

```diff
@@ -0,0 +1,48 @@
+{
+  "name": {
+    "en": "Ministry of Justice and Human Rights",
+    "local": "Ministère de la Justice et Droits Humains"
+  },
+  "url": "https://journalofficiel.cd/accueil",
```

Comment:
@siwhitehouse - I wonder whether we should link to here in the first place. http://droits-humains.gouv.cd/dh_ong_liste.html. The org you were originally trying to find an ID for is listed under IPP.... but obvs no reference or ID is published there.

Reply:
That makes sense. I hadn't seen this list before, so it's definitely worth including for people's reference